samples/monster_generated.rs

```diff
@@ -171,9 +171,56 @@ impl flatbuffers::EndianScalar for Equipment {
 }
 
 pub struct EquipmentUnionTableOffset {}
+
+#[non_exhaustive]
+#[derive(Debug, Clone, PartialEq)]
+pub enum EquipmentT {
+  NONE,
+  Weapon(Box<WeaponT>),
+}
+impl Default for EquipmentT {
+  fn default() -> Self {
+    Self::NONE
+  }
+}
+impl EquipmentT {
+  fn equipment_type(&self) -> Equipment {
```

Comment:
Why not impl Into?

Reply:
idk, `EquipmentT` and `Equipment` don't seem _that_ equivalent to me, to use `impl Into`, since one is a union and the other is just the discriminant. I'm open to bikeshedding this function though, maybe `fn discriminant` would be better since it doesn't rely on the name of union in the schema.